gladiator.js

```diff
@@ -0,0 +1,10 @@
+class gladiator(name, weapon){
+  this.name: name;
+  this.weapon: weapon;
+
+
+
+  var max = new Gladiator("Maximus", "Trident")
```

Comment:
These next three lines should not be happening inside of the class defintion. These are lines that test the implementation of your class.